qaframework-bdd-tests/src/test/resources/features/refapp-2.x/016-XssVulnerability/xssVulnerability.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Check Xss Vulnerability
+
+  Background:
+    Given user goes to active visit search page
+   
+  @selenium
+  @xssVulnerability
+  Scenario: Check XSS Vulnerability
+    When user searches for patient using patient identifier
+    And system loads  patient dashboard page of last Active visit
```

Comment:
```suggestion
    Then system loads  patient dashboard page of last Active visit
```

Reply:
did you see the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/327/files#r802017014)?

Reply:
Resolved thanks.